src/main/java/picard/util/IntervalListTools.java

```diff
@@ -629,8 +625,8 @@ static IntervalListInputType forFile(final File intervalListExtractable) {
             return INTERVAL_LIST;
         }
 
-        public static IntervalList getIntervalList(final File file, final boolean includeFiltered) {
-            return forFile(file).getIntervalListInternal(file, includeFiltered);
+        public static IntervalList getIntervalList(final PicardHtsPath file, final boolean includeFiltered) {
```

Comment:
This should probably take HtsPath instead of PicardHtsPath so it is more general.  Usuall

Reply:
Done